hal/halx86/legacy/halmps.rc

```diff
@@ -0,0 +1,5 @@
+#define REACTOS_VERSION_DLL
+#define REACTOS_STR_FILE_DESCRIPTION  "Hardware Abstraction Layer DLL"
```

Comment:
```suggestion
#define REACTOS_STR_FILE_DESCRIPTION  "x86 APIC/MPS Multiprocessor Hardware Abstraction Layer"
```

Reply:
From the point of view of compatibility with NT, all HALs have the same name ...

Reply:
Since no program looks at the file description (i.e. there won't be any compatibility problem), we can give more descriptive names to indicate for what purpose given hal is.

Reply:
@HBelusca, 
Are you 100% sure about that?
I am 100% sure that if I do the same as in NT, then there will definitely be no problems.
Is there any real use for these names?

Reply:
>  Is there any real use for these names?

I don't think there is any real use for these names, except for the user who would be better informed about what that HAL dll is about.